### DISCUSSION TITLE
Does it make sense that capitalized and non-capitalized words have different outlines? (apart from the # key)

### DISCUSSION THREAD

**Discussion by evgenyfadeev** [aerickt/lapwing-for-beginners#27] [Q&A]
corona: KROE/TPHA, KOR/KWRO/TPHA, KOR/KWROE/TPHA
Corona: #KO/RO/TPHA, #KO/ROE/TPHA, #KOR/KWRO/TPHA, #KOR/KWROE/TPHA

Is this intended or it's something that may need a correction?

**Comment by aerickt** [comment]
mhm that's just an issue on my end, thanks!

**Comment by evgenyfadeev** [comment]
maybe then it would be worth to check programmatically, because I've ran into this randomly and there may be other cases.

**Reply by aerickt** [reply]
To be honest, the proper nouns dictionary should be rewritten entirely since there have been quite a few changes to the base theory, and I never made a huge effort to keep the two dictionaries in sync. The proper nouns dictionary only has 15000 entries, so that might be something I'll do in the near future.

A script to generate proper noun outlines that are already exist in the base dictionary would be nice for sure. Could probably knock out a few thousand entries. I may look into this, but I'm only a very amateur programmer.
